go-getting-started/controllers/insert.go

```diff
@@ -0,0 +1,19 @@
+package controllers
+
+import (
+	"context"
+
+	"github.com/heroku/go-getting-started/dbinterface"
+	"github.com/heroku/go-getting-started/models"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func InsertData(collection dbinterface.CollectionAPI, joke models.Joke) (*mongo.InsertOneResult, error) {
```

Comment:
The name of the method - `InsertData` - is way too generic. It contradicts with its signature, which tells that it expect a very specific data, namely a `models.Joke`, as an argument.